dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -62,7 +62,9 @@ RUN apt-get update &&        \
         net-tools            \
 # for arm arch: Installing j2cli dependency package MarkupSafe from source relies on weeksetuptools and wheel
         python-setuptools    \
-        python-wheel
+        python-wheel         \
+# for processing json files in bash environment
+        jq
```

Comment:
>jq [](start = 8, length = 2)

We already install j2cli. Could you use that?

Reply:
The template file is used by cfggen and jq is a tool to process json formats. I wonder what the point of j2cli here?

Reply:
@qiluo-msft, it looks like you have close this issue, but it does not really get closed on the github ui. can you also do this?